karton/core/backend.py

```diff
@@ -45,11 +54,11 @@ def __init__(self, config, identity: Optional[str] = None) -> None:
         self.config = config
         self.identity = identity
         self.redis = self.make_redis(config, identity=identity)
-        self.minio = Minio(
-            endpoint=config["minio"]["address"],
-            access_key=config["minio"]["access_key"],
-            secret_key=config["minio"]["secret_key"],
-            secure=config.getboolean("minio", "secure", fallback=True),
+        self.s3 = boto3.client(
+            "s3",
+            endpoint_url=config["s3"]["address"],
```

Comment:
`secure` is not needed because `http/https` is in URI?

Reply:
yep